ci/travis_script_manylinux.sh

```diff
@@ -21,14 +21,18 @@
 set -ex
 
 pushd python/manylinux1
-docker run --shm-size=2g --rm -e PYARROW_PARALLEL=3 -v $PWD:/io -v $PWD/../../:/arrow quay.io/xhochy/arrow_manylinux1_x86_64_base:llvm-7-manylinux1 /io/build_arrow.sh
+docker run --shm-size=2g --rm \
+  -e PYARROW_PARALLEL=3 \
+  -v $PWD:/io \
+  -v $PWD/../../:/arrow \
+  quay.io/xhochy/arrow_manylinux1_x86_64_base:llvm-7-manylinux1 \
```

Comment:
Ideally We should not run the test inside the image, just outside to prevent the issues like this.

Reply:
Yes, running them somewhere else shound ensure that we have packaged all necessary things into the wheel.